mapentity/static/mapentity/mapentity.context.js

```diff
@@ -19,7 +19,9 @@ MapEntity.Context = new function() {
         // Form filters
         if (filter) {
             // exclude bbox field, since it comes from the map view.
-            var fields = $($(filter).serializeArray()).filter(function (){ return this.name != 'bbox';});
+            var fields = $($(filter).serializeArray()).filter(function (){
+                return this.name != 'bbox' && this.value !== '';
```

Comment:
Are the fields filled in by users? If so, you can filter values filled in only with spaces as follows: 
```suggestion
                return this.name != 'bbox' && this.value.trim() !== '';
```